lib/assets/javascripts/twine.coffee

```diff
@@ -462,19 +457,19 @@
 
   setupEventBinding = (eventName) ->
     Twine.bindingTypes["bind-event-#{eventName}"] = (node, context, definition) ->
-      onEventHandler = (event, data) ->
+      onEventHandler = (event) ->
```

Comment:
I think removing `data` on 472 could could potentially break consumers if they rely on it. [You can access data on custom events via `.detail`.](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/detail)

```suggestion
      onEventHandler = (event, data = event.detail) ->
```

Reply:
Done. I couldn't accept this suggestion as I had to make other changes in this method to make things tick 👍